src/scss/colors.scss

```diff
@@ -5,13 +5,13 @@ $shades-enum: (5, 10, 14, 20, 50);
 $alphas-enum: (0.18, 0.2, 0.24, 0.35, 0.5, 0.56, 0.65, 0.8);
 
 :root {
-  --primary-content--under: #000000;
-  --primary-content--under-rgb: 0, 0, 0;
-  --primary-content--over: #ffffff;
-  --primary-content--over-rgb: 0, 0, 0;
-  --primary-accent--over: #ffffff;
-  --primary-accent--under: #7c46fb;
-  --primary-accent--under-rbg: 124, 70, 251;
+  --content--under: #000000;
+  --content--under-rgb: 0, 0, 0;
+  --content--over: #ffffff;
+  --content--over-rgb: 255, 255, 255;
+  --accent--over: #ffffff;
```

Comment:
Should there be `-rbg` named constant for this as well? For symmetry 🤷

Reply:
added symmetry a9aad71bef3d1bd0c1cf71598d0fe6ddfede020f